### ISSUE TITLE
aggregation returns error with 1D frequency. '24h' frequency is ok.

### ISSUE THREAD

**Issue by cdeline** [NREL/rdtools#114]
This is weird.  When I'm running one of the Task 13 systems, I get a `ValueError: Values falls after last bin` after a call to rdtools.aggregation_insol.

The problem is fixed if I replace the `frequency = '1D'` with `frequency = '24h'`

Here's the traceback.  Recommend we change the default and notebook examples from 1D to 24h

> daily = rdtools.aggregation_insol(filtered.normalized, filtered.insolation, frequency = '1D')
> Traceback (most recent call last):
> 
>   File "<ipython-input-56-aadfc5a5167a>", line 1, in <module>
>     daily = rdtools.aggregation_insol(filtered.normalized, filtered.insolation, frequency = '1D')
> 
>   File "c:\users\cdeline\documents\python scripts\kwh_rdtools\rdtools\aggregation.py", line 23, in aggregation_insol
>     aggregated = (insolation * normalized_energy).resample(frequency).sum() / insolation.resample(frequency).sum()
> 
>   File "C:\Users\cdeline\anaconda3\lib\site-packages\pandas\core\resample.py", line 855, in f
>     return self._downsample(_method, min_count=min_count)
> 
>   File "C:\Users\cdeline\anaconda3\lib\site-packages\pandas\core\resample.py", line 999, in _downsample
>     self._set_binner()
> 
>   File "C:\Users\cdeline\anaconda3\lib\site-packages\pandas\core\resample.py", line 173, in _set_binner
>     self.binner, self.grouper = self._get_binner()
> 
>   File "C:\Users\cdeline\anaconda3\lib\site-packages\pandas\core\resample.py", line 181, in _get_binner
>     binner, bins, binlabels = self._get_binner_for_time()
> 
>   File "C:\Users\cdeline\anaconda3\lib\site-packages\pandas\core\resample.py", line 988, in _get_binner_for_time
>     return self.groupby._get_time_bins(self.ax)
> 
>   File "C:\Users\cdeline\anaconda3\lib\site-packages\pandas\core\resample.py", line 1421, in _get_time_bins
>     ax_values, bin_edges, self.closed, hasnans=ax.hasnans)
> 
>   File "pandas\_libs\lib.pyx", line 691, in pandas._libs.lib.generate_bins_dt64
> 
> ValueError: Values falls after last bin

**Comment by cwhanse** [comment]
There was a related [pandas bug](https://github.com/pandas-dev/pandas/issues/24127)  I wonder if you've found another.

**Comment by mdeceglie** [comment]
Thanks @cdeline.  It would be helpful if you could come up with the minimal copy-pastable example that shows the bug so we can reproduce it and get to the bottom of it.

**Comment by cwhanse** [comment]
what pandas version @cdeline ?  I don't reproduce this behavior with 0.24.2

**Comment by cdeline** [comment]
Example bug generation in pandas version 0.24.0.  But it appears to be fixed in pandas 0.24.2!   If you're interested, the following generates an error when using a timestamp on 10/25/2009 in CET.  Should we require pandas >= 0.24.2 in setup.py?  At the moment we only require pandas >= 0.23.0

```
import pandas as pd
data = pd.DataFrame.from_dict({'power':[0]})
data.index = pd.to_datetime(['2009-10-25 22:00:01']) # Errors any time on 10/25/2009 before 23:00
data.index = data.index.tz_localize('Europe/Paris') #Errors with CET, other timezones work
this_will_error = data.power.resample('1D').sum() # '24h' works
```
